src/test/java/com/google/sps/TimeHandlerTest.java

```diff
@@ -0,0 +1,169 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.text.ParseException; 
+import com.google.sps.data.TimeHandler;
+import org.json.JSONObject;
+
+public class TimeHandlerTest {
+
+  public static final String YEAR_2020_07_15 = "2020-07-15";
+  public static final String YEAR_2020_06_15 = "2020-06-15";
+  public static final String YEAR_5001_09_17 = "5001-09-17";
+  public static final String YEAR_2020_07_16 = "2020-07-16";
+
+  public static final String TIME_1_00 = "01:00";
+  public static final String TIME_12_00 = "12:00";
+  public static final String TIME_14_00 = "14:00";
+  public static final String TIME_5_00 = "05:00";
+  public static final String TIME_19_00 = "19:00";
+
+  public static final String PACIFIC_DAYLIGHT = "PDT";
+  public static final String CENTRAL_DAYLIGHT = "CDT";
+
+  JSONObject json = new JSONObject();
+
+  @Test
+  public void parseInputDateTimeTest() {
+    Date actualDateTime = TimeHandler.parseInputDateTime(YEAR_2020_07_15, TIME_14_00, PACIFIC_DAYLIGHT, json);
+    
+    SimpleDateFormat formatter = new SimpleDateFormat("MM-dd-yyyy HH:mm z");
+    formatter.setTimeZone(TimeZone.getTimeZone(PACIFIC_DAYLIGHT));
+    Date expectedDateTime = new Date();
```

Comment:
According to the Java docs, the `Date()` constructor "allocates a Date object and initializes it so that it represents the time at which it was allocated, measured to the nearest millisecond."

I don't think that's what we want in this test. We expect the result to be a date representing 2pm on July 15, 2020. As such expectedDateTime should be something like:

```
Date.parse("2020-07-15T14:00:00Z");
```

This holds for other tests as well.